### ISSUE TITLE
Change redaction layers

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#3704]
* As an IAO analyst
* I want to be able to switch between redaction layers
* so that (reason/ why/ value) 

**Assumptions & Scope**	
What are the assumptions for this story?

What is IN scope?

What is NOT in scope?

**Acceptance Criteria**	

**Scenario 1: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 2: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 3: xxxxxx**
...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by lmullane** [comment]
@m-prodan, @JHarrietha-AOT

Please review this story AC.

@JHarrietha-AOT, could you clarify what the Harms and Ministry Sign Off layers are for in the design?

https://jacklyn808742.invisionapp.com/console/Redaction-cla8feezv003o015h7op82c5c/cleg4xqe55wt3018ba31e3cvi/play

Cc @liseandtea, @KyEggleston , @abin-aot

**Comment by JHarrietha-AOT** [comment]
@lmullane @m-prodan @liseandtea @KyEggleston 

Changes based off of conversation at release planning
https://jacklyn808742.invisionapp.com/overview/Redaction-cla8feezv003o015h7op82c5c/screens

Desired future state - for future ticket writing
https://jacklyn808742.invisionapp.com/overview/Redaction-cla8feezv003o015h7op82c5c/screens

Let me know if you have any additional changes or if there was anything I missed from our conversation earlier

**Comment by lmullane** [comment]
@m-prodan and @liseandtea 

Is there ever a scenario where an analyst will review and change the redactions in a Response Package? Or must they make the changes in the Redline first, and then create the Response Package?

**Comment by liseandtea** [comment]
@m-prodan @lmullane Yes, there are times when an analyst has to make changes to a response package once it has been closed. Usually, this is a breach situation. However, to make the changes, the request has to be re-opened in AXIS and the records moved from the "Request Folder" back to the "Review Log" in document management.

**Comment by m-prodan** [comment]
Need to figure out a way to break this up - per refinement, this is likely to be more than a 21, but less than a 40. Need to find the right way to split this story

**Comment by m-prodan** [comment]
Possible idea is to split this story up with one story being, create response package, and the second story being the switching between layers.

**Comment by m-prodan** [comment]
From what we can see - this story is likely currently being handled with the save response package story. Switching layers is already available.
